src/applications/gi/components/profile/schoolRatings/SchoolCategoryRating.jsx

```diff
@@ -18,7 +18,7 @@ const SchoolCategoryRating = ({ titleObj, questionsObj }) => {
   };
 
   return (
-    <div className="medium-screen:vads-l-col--6 small-screen:vads-l-col--12 xsmall-screen:vads-l-col--12">
+    <div className="medium-screen:vads-l-col--6 mobile-lg:vads-l-col--12 mobile:vads-l-col--12">
```

Comment:
Hey @GovCIOLiz - It wasn't clear in my original announcement about breakpoint names and utilities, for that I apologize. The grid classes here, `vads-l-col--12` and `vads-l-col--6`, are not actually utility classes and have not been converted to the new breakpoint names yet. Could you please revert these to `small-screen` and `xsmall-screen`? They should still work as intended because they're still getting imported from formation through `core.scss`.

Reply:
I'll drop a message in vfs-frontend to communicate this.